lib/config/rules/stylistic-issues.js

```diff
@@ -64,14 +64,24 @@ module.exports = {
   'linebreak-style': 'off',
   // Enforces empty lines around comments
   'lines-around-comment': ['error', {beforeBlockComment: true}],
+  // require or disallow an empty line between class members
+  'lines-between-class-members': [
+    'error',
+    'always',
+    {exceptAfterSingleLine: true},
+  ],
   // Enforce position of line comments
   'line-comment-position': ['error', {position: 'above'}],
   // Enforce a maximum file length
   'max-lines': 'off',
+  // 	enforce a maximum number of line of code in a function
+  'max-lines-per-function': 'off',
   // Specify the maximum depth callbacks can be nested
   'max-nested-callbacks': 'off',
   // Specify the maximum number of statements allowed per line
   'max-statements-per-line': ['error', {max: 2}],
+  // enforce a particular style for multiline comments <FEEDBACK>
+  'multiline-comment-style': 'off',
```

Comment:
Was iffy on this one. Found it a bit annoying, but maybe that's just me. Curious to hear others thoughts.